src/resources/Pipelines/StatementGroups/StatementGroupsInterfaces.ts

```diff
@@ -7,17 +7,47 @@ import {
 
 export interface StatementGroupList {
     groups: StatementGroupModel[];
+
+    /**
+     * The total number of matching statements across all pages of results.
+     */
     totalCount: number;
+
     groupComposition: PipelineGroupsComposition;
 }
 
 export type StatementGroupModel = PermanentStatementGroup | CampaignStatementGroup;
 
 interface StatementGroupModelBase {
+    /**
+     * The unique identifier of the statement group.
+     */
     id: string;
+
+    /**
+     * The name of the statement groups.
+     */
     name: string;
+
+    /**
+     * The intented purpose of this statement group.
+     */
     description?: string;
+
+    /**
+     * The id of the condition that must be met by a query in order to route that query through this query pipeline.
```

Comment:
This is not valid. Maybe something like:

> The unique identifier of the condition that must be satisfied for a request to be processed by the Statement Group.

cc: @rtherien

Reply:
I like the suggestion!